### PR TITLE
bacpop-65 use constellation

### PR DESCRIPTION
**Pull request by hillalex**
Use constellation; follows pattern of hint-deploy.

### CODE REVIEW

File: config/beebop.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    repo: mrcide
+    name: beebop-server
+    tag: bacpop-65
+  port: 4000
+api:
+  image:
+    repo: mrcide
+    name: beebop-py
+    tag: main
+  db_location: ./storage/GPS_v4_references
```
Comment (richfitz) [inline config/beebop.yml:23]: For prod do we want to use the big DB Marie? Do you know if this works Alex?

**Review by richfitz — COMMENTED** [review]
Thanks, this works great. One question about the db (the big one is really big but probably what we want right?). I do wonder if kill would be generally safe here as we don't do much with a db - just redis workers, but that should all be crashsafe, and we'll end up killing workers that are doing a job anyway as they won't finish within the grace period probably

**Comment by hillalex** [comment]
> Thanks, this works great. One question about the db (the big one is really big but probably what we want right?). I do wonder if kill would be generally safe here as we don't do much with a db - just redis workers, but that should all be crashsafe, and we'll end up killing workers that are doing a job anyway as they won't finish within the grace period probably

Relatedly, at the moment the setup downloads the db each time, so one of my questions was whether we should avoid doing this (esp if the prod one take a while)

**Comment by richfitz** [comment]
we should definitely avoid redownloading if the db exists, need not be done here though

File: config/beebop.yml (hunk 1 above)
Comment (hillalex) [reply config/beebop.yml:23]: Oh, in fact, we're missing a config option here, this actually doesn't control which db to download

Comment (hillalex) [reply config/beebop.yml:23]: Have updated to make this configurable now

**Review by richfitz — APPROVED** [review]

**Pull request merged**